ideascube/conf/kb_fra_crfaix.py

```diff
@@ -0,0 +1,13 @@
+"""KoomBook conf"""
+from .kb import *  # pragma: no flakes
+
+LANGUAGE_CODE = 'fr'
+IDEASCUBE_NAME = 'CRF AIX'
```

Comment:
The commit message only says you want to remove Kalite from the home, but then you define these two above.

Are they necessary?